src/Sentry/SentrySdk.cs

```diff
@@ -147,21 +217,36 @@ public void Dispose()
         /// <param name="state">A state object to be added to the scope</param>
         /// <returns>A disposable that when disposed, ends the created scope.</returns>
         [DebuggerStepThrough]
-        public static IDisposable PushScope<TState>(TState state) => _hub.PushScope(state);
+        public static IDisposable PushScope<TState>(TState state)
+        {
+            // pushScope is no-op in global hub mode
+            if (!_globalHudMode)
+            {
+                return GetCurrentHub().PushScope(state);
+            }
+            return null;
```

Comment:
Returning `null` for `IDisposable` is a bad idea. `using(SentrySdk.PushScope)` will throw.

Reply:
@bruno-garcia  The correct way would be to not return anything (changing from IDisposable to void) but afraid that will result in some breaking change.

Reply:
```
    internal class FalseDisposable : IDisposable
    {

        public void Dispose() { }
    }

```
creating a static class from FalseDisposable inside of SentrySDK and only returning it on Push/Pop scope only if the global mode is true, seems to be an "OK" solution to avoid breaking changes (note that if not using global mode as most .Net application does, it'll work the same previously)

Reply:
yeah we use this approach elsewhere with a `private class`, I can't think of a better way, sadly